snap/snapcraft.yaml

```diff
@@ -292,8 +293,22 @@ parts:
       export CGO_LDFLAGS="-L${CRAFT_STAGE}/lib/ -L${CRAFT_STAGE}/usr/local/lib/"
       export CGO_LDFLAGS_ALLOW="(-Wl,-wrap,pthread_create)|(-Wl,-z,now)"
 
+      # Determine OVN package version
+      pkg_version=$(
+          apt-cache policy ovn-host | \
+          awk '/Candidate:/{ print $2 }' | \
+          sed -rne 's/([0-9.]+)[-+].*$$/\1/p'
+      )
+      git_version=$(
```

Comment:
Instead of doing all this twice, should we perhaps instead make the `microovn` part build after `ovn`, and point `adopt-info` to the `microovn` part?

Reply:
Oh, I totally misunderstood the meaning of this comment the first time i read it. I see what you mean, you want to move setting of the snap's version to this part instead of `ovn`.
Yeah, that make's sense. I'll mark this as WIP until I apply this change.

Reply:
Updated. Snap's version is now set in `microovn` part.